mc/instr/__init__.py

```diff
@@ -622,8 +625,11 @@ def encode(self, opcode, encoder, addr):
         return opcode
 
     def render(self, addr):
+        register_list = ['FB', 'SB', 'A1', 'A0', 'R3', 'R2', 'R1', 'R0']
```

Comment:
I meant that the natural order of registers is R0, R1, ..., FB, you know, as written in the ISA manual intro.

Reply:
i even checked to make sure the order was right, not noticing i was looking at popm rather than pushm, ugh. done.